Packs/CommonScripts/Scripts/Ping/Ping.yml

```
@@ -5,7 +5,7 @@ args:
   name: address
   predefined: []
   required: true
-comment: Pings an IP or url address, to verify it's up
+comment: Pings an IP or url address, to verify it's up, on xsoar8 and xsiam it can be used only on custom engines.
```

```suggestion
comment: Pings an IP or url address, to verify it's up. Note: On Cortex XSOAR 8 and Cortex XSIAM, the script can run only on a custom engine.
```
please update the readme as well

---

the script doesn't have a read me :/